robot/impl/local_robot_test.go

```diff
@@ -1320,6 +1341,10 @@ func TestStatusRemote(t *testing.T) {
 		err = decoder.Decode(status.Status)
 		test.That(t, err, test.ShouldBeNil)
 		test.That(t, convMap, test.ShouldResemble, armStatus)
+
+		// Test that LastReconfigured values are from remotes, and not set based on
+		// when local resource graph nodes were added for the remote resources.
+		test.That(t, status.LastReconfigured, test.ShouldEqual, lastReconfigured)
```

Comment:
Added this assertion in to show that `lastReconfigured` values are only calculated by the robot owning the resource, and not by when the resource was added to the main robot's resource graph.